src/WebJobs.Script/Utility.cs

```diff
@@ -985,17 +985,27 @@ public static bool IsMediaTypeOctetOrMultipart(MediaTypeHeaderValue mediaType)
             }
         }
 
-        // EnableWorkerIndexing set through AzureWebjobsFeatuerFlag always take precdence
-        // if AzureWebjobsFeatuerFlag is not set then WORKER_INDEXING_ENABLED hosting config controls stamplevel enablement
-        // if WORKER_INDEXING_ENABLED is set and WORKER_INDEXING_DISABLED contains the customers app name worker indexing is then disabled for that customer only
-        // Also Worker indexing is disabled for Logic apps
+        // Worker indexing is disabled for Logic apps
+        // EnableWorkerIndexing set through AzureWebjobsFeatuerFlag always take precedence
+        // If AzureWebjobsFeatuerFlag is not set and
+        // WORKER_INDEXING_DISABLED contains the customers app name worker indexing is then disabled for that customer only
         public static bool CanWorkerIndex(IEnumerable<RpcWorkerConfig> workerConfigs, IEnvironment environment, FunctionsHostingConfigOptions functionsHostingConfigOptions)
         {
             string appName = environment.GetAzureWebsiteUniqueSlotName();
-            bool workerIndexingEnabled = FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagEnableWorkerIndexing, environment)
-                                          || (functionsHostingConfigOptions.WorkerIndexingEnabled
-                                          && !functionsHostingConfigOptions.WorkerIndexingDisabledApps.ToLowerInvariant().Split("|").Contains(appName)
-                                           && !environment.IsLogicApp());
+
+            bool workerIndexingEnabled;
+            bool featureFlagSet = FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagEnableWorkerIndexing, environment);
+            workerIndexingEnabled = featureFlagSet && !environment.IsLogicApp();
```

Comment:
nit: you can just define it here, I don't think it needs to be it's own line
```suggestion
            bool workerIndexingEnabled = featureFlagSet && !environment.IsLogicApp();
```

Reply:
right sorry, i changed this so many times. forgot to move it back to its own line